File: Basic Coding Problems/01 Fundamentals/013 Write a program to enter two angles of a triangle and find the third angle/Java/triangle_angles.java
import java.util.Scanner;

public class triangle_angles {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the first angle: ");
        int firstAngle = scanner.nextInt();

        System.out.print("Enter the second angle: ");
        int secondAngle = scanner.nextInt();

        int thirdAngle = 180 - (firstAngle + secondAngle);

        System.out.println("The third angle of the triangle is: " + thirdAngle);
    }
}

File: Basic Coding Problems/01 Fundamentals/013 Write a program to enter two angles of a triangle and find the third angle/C/triangle_angles.c
#include <stdio.h>

int main() {
    int first_angle, second_angle, third_angle;

    printf("Enter the first angle: ");
    scanf("%d", &first_angle);

    printf("Enter the second angle: ");
    scanf("%d", &second_angle);

    third_angle = 180 - (first_angle + second_angle);

    printf("The third angle of the triangle is: %d\n", third_angle);

    return 0;
}